src/lib/y2partitioner/widgets/disk_page.rb

```diff
@@ -0,0 +1,65 @@
+require "cwm/widget"
+require "cwm/tree_pager"
+
+require "y2partitioner/widgets/blk_devices_table"
+require "y2partitioner/widgets/disk_description"
+require "y2partitioner/icons"
+
+module Y2Partitioner
+  module Widgets
+    class DiskPage < CWM::Page
+      def initialize(disk)
+        @disk = disk
+        self.widget_id = "disk:" + disk.name
+      end
+
+      def label
+        @disk.sysfs_name
+      end
+
+      def contents
+        icon = Icons::SMALL_ICONS_PATH + Icons::HD
+        VBox(
+          Left(HBox(
+            Image(icon, ""),
+            Heading(_("Hard Disk: ") + @disk.name)
```

Comment:
The `format` approach should be used also here:

```ruby
# TRANSLATORS: %s is a disk name
format(_("Hard Disk: %s"), @disk.name)
```

The `%s` tag helps the translators to move the added text, this is probably most important in the RTL languages.